<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModificationProduitClass</class>
 <widget class="QMainWindow" name="ModificationProduitClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ModificationProduit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="GestionStock.qrc">
    <normaloff>:/MenuPrincipal/img/modifier-les-informations.png</normaloff>:/MenuPrincipal/img/modifier-les-informations.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="header" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #0A043C;</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="verticalWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>80</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnRetour">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>80</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>80</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255,0);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="GestionStock.qrc">
              <normaloff>:/AjoutProduit/img/retour.png</normaloff>:/AjoutProduit/img/retour.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>90</width>
              <height>90</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="horizontalWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>80</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>80</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>80</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/SuppressionProduit/img/Suppression de produit.png);
margin: 5%;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Andalan</family>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #ffffff;</string>
            </property>
            <property name="text">
             <string>MODIFICATION  DE PRODUIT</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="box" native="true">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>80</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="main" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: #7FA6C8;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>65</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>65</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>45</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Aveny T WEB</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Recherche de produit</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="minimumSize">
             <size>
              <width>440</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>440</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLineEdit" name="leRecherche">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>440</width>
               <height>40</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>IBeamCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">border-radius: 10px;
background-color: #ffffff;
padding-left: 10px;</string>
             </property>
             <property name="placeholderText">
              <string>Recherche de numéro de produit ...</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btnRecherche">
             <property name="geometry">
              <rect>
               <x>400</x>
               <y>0</y>
               <width>40</width>
               <height>40</height>
              </rect>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgba(255, 255, 255,0);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="GestionStock.qrc">
               <normaloff>:/AjoutProduit/img/loupe.png</normaloff>:/AjoutProduit/img/loupe.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="verticalWidget_2" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>125</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>125</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QWidget" name="horizontalWidget" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="leftMargin">
              <number>9</number>
             </property>
             <property name="topMargin">
              <number>5</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="QWidget" name="verticalWidget" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_12"/>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="minimumSize">
                <size>
                 <width>125</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Aveny T WEB</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Numéro du produit</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lbNumProduit">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>40</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: #fff;
border-radius: 5px;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="verticalWidget_2" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_13"/>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="horizontalWidget" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="topMargin">
              <number>5</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="QWidget" name="verticalWidget" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_5"/>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="minimumSize">
                <size>
                 <width>125</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Aveny T WEB</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Nom du produit</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lbNomProduit">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>40</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: #fff;
border-radius: 5px;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="verticalWidget_2" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_6"/>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="verticalWidget_3" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: #D1DDD1;
border-radius: 10px</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>90</number>
          </property>
          <item>
           <widget class="QWidget" name="verticalWidget_4" native="true">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>120</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>40</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Couture</family>
                 <pointsize>12</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>ENTRER LE NOUVEAU NOM DU PRODUIT</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>60</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>60</height>
                </size>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QWidget" name="verticalWidget" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>225</width>
                    <height>0</height>
                   </size>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_11"/>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="leNouvProduit">
                  <property name="minimumSize">
                   <size>
                    <width>400</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>400</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: #ffffff;
border-radius: 5px;
padding-left: 15px;
font: 12pt &quot;Segoe UI&quot;;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="verticalWidget" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>225</width>
                    <height>0</height>
                   </size>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_14"/>
                 </widget>
                </item>
               </layout>
               <zorder>verticalWidget</zorder>
               <zorder>verticalWidget</zorder>
               <zorder>leNouvProduit</zorder>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="verticalWidget_5" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>80</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>80</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="verticalWidget" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_9"/>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnValider">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Aveny T WEB</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">#btnValider{  
color: #EFE8E8;
background-color: #935057;
border-radius: 10px;
}

#btnValider:hover {
  background-color: #9F8772;
  color: white;
  
}
</string>
               </property>
               <property name="text">
                <string>VALIDER</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnSuivant">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Aveny T WEB</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">#btnSuivant{  
color: #EFE8E8;
background-color: #935057;
border-radius: 10px;
}

#btnSuivant:hover {
  background-color: #9F8772;
color: white;
  
}
</string>
               </property>
               <property name="text">
                <string>SUIVANT</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="verticalWidget_2" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_15"/>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GestionStock.qrc"/>
 </resources>
 <connections/>
</ui>
